src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs

```diff
@@ -30,9 +35,20 @@ internal JsonClassInfo GetOrAddClass(Type classType)
             return result;
         }
 
+        /// <summary>
+        /// The <see cref="JsonReaderOptions"/> used when deserializing an object or value type.
```

Comment:
Should we avoid the serializing/deserializing terms and talk in terms of the APIs themselves? Otherwise, the user has to keep mapping between serialize/deserialize and read/write/parse.

Reply:
Changed to `Options to control the <see cref="Utf8JsonReader"/`